src/gc/gc.cpp

```diff
@@ -33623,6 +33620,8 @@ HRESULT GCHeap::Initialize ()
     }
 
     g_gc_pFreeObjectMethodTable = GCToEEInterface::GetFreeObjectMethodTable();
+    g_num_processors = GCToOSInterface::GetCurrentProcessCpuCount();
```

Comment:
>GetCurrentProcessCpuCount [](start = 40, length = 25)

this should not be obtained from GetCurrentProcessCpuCount which gives you how many CPUs this process is affinitized to. You could be on a 64-CPU system and the process could be affinitized to 2 of them. This should be for the whole machine (ie, in win32 it should be from GetSystemInfo).